frontend/src/components/schedule-view/events.tsx

```diff
@@ -106,12 +106,12 @@ export const Submission = ({
 
 const getItemUrl = (item: Item) => {
   if (item.type === "submission") {
-    return `/[lang]/talk/[slug]`;
+    return `/talk/[slug]`;
   }
 
   // TODO: check tbd
   if (item.type === "keynote") {
-    return `/[lang]/keynote/[slug]`;
+    return `/talk/[slug]`;
   }
```

Comment:
the keynote page doesn't exist, but it is a good idea. For now I added a link to the talk page

writing this down because this also implies that the keynote will be a submission in BE